repository.md

```diff
@@ -12,7 +12,7 @@ Using a version control system allows you to easily track changes in your softwa
 
 ## Help me choose
 
-Git is the most feature-rich, most modern and most popular by a good margin, and we heartily recommend you use it for all your version control needs. To get the best out of Git, use it in combination with GitHub.com, Bitbucket.org, or GitLab.com.
+Git is the most feature-rich, most modern and most popular version control system by a good margin, and we heartily recommend you use it for all your version control needs. To get the best out of Git, use it in combination with GitHub.com, Bitbucket.org, or GitLab.com.

```

Comment:
```suggestion
Git is the version control system which is most feature-rich, most modern and most popular by a good margin, and we heartily recommend you use it for all your version control needs. To get the best out of Git, use it in combination with GitHub.com, Bitbucket.org, or GitLab.com.

```

Reply:
Thank you @c-martinez that PR looks good. I proposed one small change to make a sentence more readable.